packages/labs/ssr/src/lib/render-value.ts

```diff
@@ -489,7 +516,49 @@ const getTemplateOpcodes = (result: TemplateResult) => {
           ops.push({
             type: 'custom-element-shadow',
           });
+        } else if (
+          !hydratable &&
+          /^(title|textarea|script)$/.test(node.tagName)
```

Comment:
Should `style` be added here. (It's also handled as a raw text element by `lit-html`)?

Reply:
Good point! Treating similarly to `<script>`, with unsafeHTML as the escape hatch.